Test/FMI3/fmi3_capi_basic_test.cpp

```diff
@@ -21,18 +21,6 @@
 #include "config_test.h"
 #include "fmilib.h"
 
-typedef struct {
-    fmi3_import_t* fmu; /* fmi3_log_forwarding is based on the fmu instance */
-} fmi3_inst_env_t;
-
-
-/* The following dummy types/functions are used only for testing in order to be able
-    invoke the necessary functions, for example FMU instantiation.
-*/
-void fmi3_dummy_log_callback(fmi3_instance_environment_t instEnv, fmi3_status_t status,
-        fmi3_string_t category, fmi3_string_t message) {
-    fmi3_log_forwarding(((fmi3_inst_env_t*)instEnv)->fmu, status, category, message);
-}
```

Comment:
Unnecessary complexity, and is not what we test, so just removed. The log callback is tested elsewhere.